### PR TITLE
[RFR] Add detail link on columns

### PR DESCRIPTION
**Pull request by manuquentin**
- [x] Add detail links in columns
- [x] Add tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
   },
   "jest": {
     "scriptPreprocessor": "<rootDir>/jest-preprocessor.js",
+    "setupEnvScriptFile": "<rootDir>/jest-bootstrap.js",
```
Comment (jeromemacias) [inline package.json]: No need for this one with #36.

File: app/Component/Column/BooleanColumn.js [hunk 2]
```diff
@@ -2,7 +2,13 @@ import React from 'react';
 
 class BooleanColumn extends React.Component {
     render() {
-        let className = !!this.props.value ? 'boolean-true' : 'boolean-false';
+        let {value, detailAction} = this.props,
+            className = !!value ? 'boolean-true' : 'boolean-false';
```
Comment (jeromemacias) [inline app/Component/Column/BooleanColumn.js]: Can you write one `let` declaration per line please ?

**Comment by jeromemacias** [comment]
Need rebase

File: e2e/listViewSpec.js [hunk 3]
```diff
@@ -111,5 +111,13 @@ testMethod('ListView', function () {
                 expect(browser.getCurrentUrl()).toContain('/posts/edit/1');
             });
         });
+
+        describe('detail link', function() {
+            it('should go to edit view', function() {
+                // Click on first detail link
+                $('table tr:nth-child(1) td:nth-child(2) a').click();
+                browser.sleep(500);
```
Comment (jeromemacias) [inline e2e/listViewSpec.js]: Should'nt it test something ?

Comment (manuquentin) [inline e2e/listViewSpec.js]: Assertion added

**Pull request merged**